emission/core/wrapper/confirmedplace.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+from future import standard_library
+standard_library.install_aliases()
+from builtins import *
+import emission.core.wrapper.place as ecwp
+import emission.core.wrapper.wrapperbase as ecwb
+
+class Confirmedplace(ecwp.Place):
+    props = ecwp.Place.props
+    props.update({"raw_place": ecwb.WrapperBase.Access.WORM,
+                  "cleaned_place": ecwb.WrapperBase.Access.WORM,
+                  "inferred_labels": ecwb.WrapperBase.Access.WORM,
+                  "expectation": ecwb.WrapperBase.Access.WORM,
```

Comment:
as we discussed, we are not planning to run label inferences on place right now since we don't have a use case for it, so we can remove these.

Reply:
please remove the others below as well

Reply:
Removed these in https://github.com/shankari/e-mission-server/pull/4/commits/64f1f5153ac795255174f39240cfddb7568a7bd2

I removed the following properties in `confirmed_place`
- inferred_labels
- expectation
- inferred_primary_mode
- user_input

Because we don't need inferred labels and expectation, and we also don't need a primary mode at a place, or a 1:1 user input at a place either

I did leave `additions` because we want this for the time-use and missing-trip surveys